### PR TITLE
Task7

### PR DESCRIPTION
**Pull request by dvoryak**

### CODE REVIEW

File: src/app/components/header/header.component.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
             <ng-template #loggedOut>
                 <a
                         class="header__link"
-                        (click)="onLogIn()">
+                        (click)="onLogIn()" routerLink="login">
```
Comment (ScreamCat1) [inline src/app/components/header/header.component.html]: In this case, we are logged in,  and navigate to the `link` page. please delete `logged` functionality and do it after submitting form

File: src/app/components/course-page/course-page.component.ts [hunk 2]
```diff
@@ -21,13 +22,16 @@ export class CoursePageComponent implements OnInit {
...
     this._courses = this.courseService.getCourses();
   }
 
-  public onCreateCourse(): void {}
+  public onCreateCourse(): void {
+    console.log('Create course event initiated');
+    this.router.navigateByUrl('/create-course');
+  }
 
   public onEdit(): void {}
```
Comment (ScreamCat1) [inline src/app/components/course-page/course-page.component.ts]: Should be implemented

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -1,8 +1,37 @@
...
+  },
+  {
+    path: 'create-course',
+    component: AddCoursePageComponent,
+    data: {
+      breadcrumb: 'Create'
+    }
+  },
+  {path: '', redirectTo: '/courses', pathMatch: 'full'},
+  {path: '**', component: NotFoundComponent},
```
Comment (ScreamCat1) [inline src/app/app-routing.module.ts]: We should have  `not-found` or `404` url
So you need redirect from unexpected url to new

File: src/app/app-routing.module.ts [hunk 4]
```diff
@@ -1,8 +1,37 @@
...
+import {AddCoursePageComponent} from './components/add-course-page/add-course-page.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: 'login',
+    component: LoginPageComponent,
+    data: {
+      breadcrumb: 'Login'
```
Comment (ScreamCat1) [inline src/app/app-routing.module.ts]: I don't see places when you use it

File: src/app/components/header/header.component.html (hunk 1 above)
Comment (dvoryak) [reply src/app/components/header/header.component.html]: The auth logic was rewritten in the task8

**Pull request merged**